tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -155,6 +155,22 @@ def test_exponential_scheduler(max_epochs, initial_value, gamma):
     exp_state_parameter_scheduler.load_state_dict(state_dict)
 
 
+@pytest.mark.parametrize(
+    "max_epochs, initial_value, gamma", [(3, 10, 0.99)],
+)
```

Comment:
I don’t understand what is exactly tested. Maybe it is worth to add some assert. What do you think ?

Reply:
Thanks for your comments @sdesrozis  , just wanted to test that no KeyError exception was raised, will push an update to the PR.